### PR TITLE
filedelete: fix stop plug-in

### PR DESCRIPTION
**Pull request by skvl**

### CODE REVIEW

File: src/plugins/filedelete/filedelete.cpp [hunk 1]
```diff
@@ -1449,12 +1449,21 @@ filedelete::filedelete(drakvuf_t drakvuf, const filedelete_config* c, output_for
 filedelete::~filedelete()
 {
     stop();
+    /* NOTE One should rund drakvuf loop to restore VM state.
+     *
+     * The plug-in injects syscalls thuns changes the state. So to avoid BSOD
```
Comment (tklengyel) [inline src/plugins/filedelete/filedelete.cpp:1454]: Typo: thus

File: src/plugins/filedelete/filedelete.cpp [hunk 2]
```diff
@@ -1449,12 +1449,21 @@ filedelete::filedelete(drakvuf_t drakvuf, const filedelete_config* c, output_for
...
+     * one should restore state here. This requires to allow VM to live for a
+     * while.
+     *
+     * Hint: there is no need to waite all files read finish. Just waite every
```
Comment (tklengyel) [inline src/plugins/filedelete/filedelete.cpp:1458]: Typo: wait

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/plugins/filedelete/filedelete.cpp (hunk 1 above)
Comment (skvl) [reply src/plugins/filedelete/filedelete.cpp:1454]: Thank you

File: src/plugins/filedelete/filedelete.cpp (hunk 2 above)
Comment (skvl) [reply src/plugins/filedelete/filedelete.cpp:1458]: Thank you

**Pull request merged**